app/controllers/api/v1/consignments_controller.rb

```diff
@@ -6,15 +6,15 @@ class ConsignmentsController < ApiController
       def index
         authorize! :read, Consignment
         excluded_columns = %w[created_at updated_at]
-        consignment_api_columns = Consignment.attribute_names.reject do |column|
-          excluded_columns.include? column
-        end
-        render json: Consignment.select(consignment_api_columns), include: included_params
+        consignment_api_columns = Consignment.attribute_names - excluded_columns
+        consignments_data = paginate_collection(Consignment.select(consignment_api_columns))
+        render json: { consignments: consignments_data[0].to_json(include: included_params),
+                       consignments_count: consignments_data[1][:total_count] }
```

Comment:
```suggestion
        consignments, meta = paginate_collection(Consignment.select(consignment_api_columns))
        render json: { consignments: consignments.to_json(include: included_params),
                                consignments_count: meta.total_count }
```

старайтесь чтоб код выглядел проще